spec/dummy/app/views/components/modal.slim

```diff
@@ -0,0 +1,6 @@
+button#show Show
+
+= ui_component 'modal', \
```

Comment:
But having a ui_component change the behaviour of arbitrary other elements in the page is better?